res/controllers/Traktor-Kontrol-S3-hid-scripts.js

```diff
@@ -1522,14 +1521,14 @@ TraktorS3.Deck = class {
     }
 
     lightHotcue(number) {
-        const loaded = engine.getValue(this.activeChannel, "hotcue_" + number + "_status");
-        const active = engine.getValue(this.activeChannel, "hotcue_" + number + "_activate");
-        let ledValue = this.controller.hid.LEDColors.WHITE;
-        if (loaded) {
+        const hotCueStatus = engine.getValue(this.activeChannel, `hotcue_${  number  }_status`);
+        let ledValue = this.controller.hid.LEDColors.OFF;
+        // If hotcue is set, change the color.
```

Comment:
please restore the default dim WHITE for unset cues

Reply:
and I'm used to them being on.  How to resolve the dispute?  Fundamentally there is no "correct" answer, it's a matter of taste.  In the past, we have deferred to the person who owns the controller, which would be me in this case.

Reply:
happy to do so.  thank you!

Reply:
Yup that works great!  And I'm fine if the default hotcue color is Black (OFF)

Reply:
Great. Thanks I'll change it to the mixxx default then.